upload/tokenless/github_actions.py

```diff
@@ -111,19 +114,6 @@ def verify(self):
             finish_time_with_buffer = build_finish_date_obj + timedelta(minutes=10)
             now = datetime.utcnow()
             if not now <= finish_time_with_buffer:
-                log.warning(
-                    "Actions workflow run is stale",
-                    extra=dict(
-                        build=build,
-                        commit=self.upload_params.get("commit"),
-                        finish_time_with_buffer=finish_time_with_buffer,
-                        job=self.upload_params.get("job"),
-                        now=now,
-                        owner=self.upload_params.get("owner"),
-                        repo_name=self.upload_params.get("repo"),
-                        time_diff=now - finish_time_with_buffer,
-                    ),
-                )
```

Comment:
removed this because it looked like the exact same warning was being logged twice, seems like an accident.